src/GraphQL/Types/Composite/AutoRegisteringOutputHelper.cs

```diff
@@ -10,7 +10,7 @@ namespace GraphQL.Types;
 internal static class AutoRegisteringOutputHelper
 {
     private static readonly IFieldResolver _invalidFieldResolver = new FuncFieldResolver<object?>(_ => throw new InvalidOperationException("This field resolver should never be called. It is only used to prevent the default field resolver from being used."));
```

Comment:
```suggestion
```

Reply:
Another option would be to have these 'invalid resolvers' set by the schema validator to ensure that interface resolvers are not used.

Reply:
I understand. I would prefer remove.